README.md

```diff
@@ -38,43 +23,59 @@ events (e.g. all data has been exhausted), the threshold property in question (e
 `lowerThreshold`) may be set to a falsy value to disable events and clear the associated
 triggered property.
 
-### Example
+See: [Documentation](https://www.webcomponents.org/element/@polymer/iron-scroll-threshold),
+  [Demo](https://www.webcomponents.org/element/@polymer/iron-scroll-threshold/demo/demo/index.html).
 
-```html
-<iron-scroll-threshold on-lower-threshold="loadMoreData">
-  <div>content</div>
-</iron-scroll-threshold>
+## Usage
+
+### Installation
+```
+npm install --save @polymer/iron-scroll-threshold
 ```
 
+### In a Polymer 3 element
```

Comment:
no "in an html file" section? Probably just have to use an `addEventListener` instead.